libs/header/p2p/options.go

```diff
@@ -120,26 +101,14 @@ func WithNetworkID[T parameters](networkID string) Option[T] {
 }
 
 // ClientParameters is the set of parameters that must be configured for the exchange.
-// TODO: #1667
 type ClientParameters struct {
-	// the target minimum amount of responses with the same chain head
-	MinResponses int
-	// MaxRequestSize defines the max amount of headers that can be handled at once.
-	MaxRequestSize uint64 // TODO: Rename to MaxRangeRequestSize
-	// MaxHeadersPerRequest defines the max amount of headers that can be requested per 1 request.
-	MaxHeadersPerRequest uint64 // TODO: Rename to MaxHeadersPerRangeRequest
-	// MaxAwaitingTime specifies the duration that gives to the disconnected peer to be back online,
-	// otherwise it will be removed on the next GC cycle.
-	MaxAwaitingTime time.Duration
-	// DefaultScore specifies the score for newly connected peers.
-	DefaultScore float32
-	// RequestTimeout defines a timeout after which the session will try to re-request headers
+	// MaxHeadersPerRangeRequest defines the max amount of headers that can be requested per 1 request.
+	MaxHeadersPerRangeRequest uint64
+	// RangeRequestTimeout defines a timeout after which the session will try to re-request headers
 	// from another peer.
-	RequestTimeout time.Duration // TODO: Rename to RangeRequestTimeout
-	// TrustedPeersRequestTimeout a timeout for any request to a trusted peer.
-	TrustedPeersRequestTimeout time.Duration
-	// MaxTrackerSize specifies the max amount of peers that can be added to the peerTracker.
-	MaxPeerTrackerSize int
+	RangeRequestTimeout time.Duration
+	// TrustedPeersRangeRequestTimeout a timeout for any request to a trusted peer.
+	TrustedPeersRangeRequestTimeout time.Duration
```

Comment:
Why is this RangeRequest? We only have special single header requests to trusted peers, correct?

Reply:
oops. It seems that `variable refactoring` in Goland changed this one too. Thanks